packages/type-safe-api/scripts/type-safe-api/generators/java/templates/server/operationConfig.ejs

```diff
@@ -0,0 +1,30 @@
+
+###TSAPI_WRITE_FILE###
+{
+  "dir": "<%- metadata.srcDir %>/api/operation_config",
+  "name": "OperationConfig",
+  "ext": ".java",
+  "overwrite": true
+}
+###/TSAPI_WRITE_FILE###package <%- metadata.packageName %>.api.operation_config;
+
+import <%- metadata.packageName %>.model.*;
+
+import java.util.HashMap;
+import java.util.Map;
+
+// Generic type for object "keyed" by operation names
+@lombok.Builder @lombok.Getter
+public class OperationConfig<T> {
+    <%_ allOperations.forEach((operation) => { _%>
+    private T <%- operation.name %>;
+    <%_ }); _%>
+
+    public Map<String, T> asMap() {
+        Map<String, T> map = new HashMap<>();
```

Comment:
I'm a little rusty on Java generics; is this <> syntax valid?

Reply:
[Yes!](https://docs.oracle.com/javase/tutorial/java/generics/types.html?ref=sangkon.com#:~:text=In%20Java%20SE%207%20and,type%20arguments%20from%20the%20context.) 😄